modules/statics/src/base.ts

```diff
@@ -2670,7 +2670,7 @@ export enum UnderlyingAsset {
   'bsc:zig' = 'bsc:zig',
   'bsc:eden' = 'bsc:eden',
   'bsc:m' = 'bsc:m',
-  'bsc:cash+' = 'bsc:cash+',
+  'bsc:cashplus' = 'bsc:cashplus',
```

Comment:
Is the "+" sign a problem for our systems. That is the official symbol. Giving it a different symbol if supported by the underlying architecture at BitGo may mislead customers and make it harder to search for.

<img width="283" height="75" alt="Screenshot 2025-09-22 at 3 02 17 PM" src="https://github.com/user-attachments/assets/1e5ef68f-21a8-4599-ac1d-c00ed48ebe5a" />